assets/base.css

```diff
@@ -982,6 +982,10 @@ summary::-webkit-details-marker {
   }
 }
 
+.grid:last-child {
+  margin-bottom: 0;
+}
```

Comment:
This accounts for any case where `.grid` is the last element - looking through the code it seems like this is often the case.  The only thing that it does cause, which might be an issue, is that if it's the last child of some parent element that has something else after it it might not be great.

I'm going to go through and see if that's the case anywhere.  I notice that it does squish the margin between the cards in Featured Collection and the View all button.

Reply:
We could look into why the grid component (which is used in a variety of scenarios) imposes a `2rem` bottom margin in the first place. I'd be in favour of setting `.grid` to a default zero top and bottom margin and only add margin in cases where it's really needed. That would require testing each scenario, so probably out of scope for this PR.